src/main/java/org/mskcc/cbio/portal/scripts/ImportTimelineData.java

```diff
@@ -135,8 +135,13 @@ public void run() {
         try {
 		    String description = "Import 'timeline' data";
 
-		    OptionSet options = ConsoleUtil.parseStandardDataAndMetaOptions(args, description, false);
-		    String dataFile = (String) options.valueOf("data");
+		    OptionSet options = ConsoleUtil.parseStandardDataAndMetaOptions(args, description, true);
+			if (options.has("loadMode") && !"bulkLoad".equals(options.valueOf("loadMode"))) {
+				throw new UnsupportedOperationException("This loader supports bulkLoad load mode only, but "
+						+ options.valueOf("loadMode")
+						+ " has been supplied.");
+			}
+			String dataFile = (String) options.valueOf("data");
```

Comment:
was this message meant for src/main/java/org/mskcc/cbio/portal/scripts/ImportCopyNumberSegmentData.java ?

Reply:
see here https://github.com/cBioPortal/cbioportal-core/pull/47/commits/95c32f8eb3abb84c2d6d2deb9932dfb072c31786 and here https://github.com/cBioPortal/cbioportal-core/pull/47/commits/fc84a410c540be471258697a746b598d507246b8

Reply:
I see, apparently https://github.com/cBioPortal/cbioportal-core/blob/main/src/main/java/org/mskcc/cbio/portal/scripts/ImportTimelineData.java is also always bulkload mode.